rails-commenting.rb

```diff
@@ -7,56 +7,59 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
-class BlogPostsController < ApplicationController
+class BlogPostsController < ApplicationController # BlogPostsController is the class created that is inheriting from the applicationController, which is allowing it to take and use its behaviors and functionality. Inside the application controller you can put different code functionality's such as before/after actions, helper methods and json renders or more. and this is the main source that all of the other controllers inherit from.
   def index
     # ---2)
-    @posts = BlogPost.all
+    @posts = BlogPost.all # I am assuming index was supposed to be under line 2 as well so I'll expain it here, index represents the R in CRUD for read. it will list all of the items in that specified model, so for this example, its saying list all of the posts from the blog post model. I try to remember this from our examples last week as the 'home page' where all of your blogs were displayed and you could click on any of them to perform the other actions.Its associated HTTP verb is GET, and the index routing is get '/resources' => 'resources#index'
```

Comment:
Nope, just the instance variable but you are correct, this will display all BlogPost!